code/modules/events/event.dm

```diff
@@ -5,11 +5,12 @@
 	var/min_weight	= 0 // The minimum weight that this event will have. Only used if non-zero.
```

Comment:
All these except `weight_mod` and `enabled` don't have to be set due to them being instantly overriden in the New